src/types.rs

```diff
@@ -989,11 +989,9 @@ pub trait FromRedisValue: Sized {
             .collect())
     }
 
-    /// This only exists internally as a workaround for the lack of
-    /// specialization.
-    #[doc(hidden)]
+    /// Convert bytes to a single element vector.
     fn from_byte_vec(_vec: &[u8]) -> Option<Vec<Self>> {
-        None
```

Comment:
Assume that we have a variable like this `let values: Vec<i32> = conn.get(keys)`.
If `keys` is a single element vector, it will use `redis.cmd("GET")` which might return `Value::Data`.
And when the return data is `Value::Data`, it will raise an error at: https://github.com/redis-rs/redis-rs/blob/470d7c58713e7c39e65ea2e84c0e5ae2abcc147d/redis/src/types.rs#L1166-L1169

This PR tried to parse a `Value::Data` to a single element vector instead of raising an error.